compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/internal/ClipboardUtils.kt

```diff
@@ -43,3 +44,12 @@ internal expect fun AnnotatedString?.toClipEntry(): ClipEntry?
  * Otherwise, it returns false.
  */
 internal expect fun ClipEntry?.hasText(): Boolean
+
```

Comment:
These are the common changes. Are they critical? See https://docs.google.com/document/d/15vffH9b_8Jqy4I9RK103HgCwr1c8lhhm1MGT7pvfT8w/edit?tab=t.0 for the poliucy

Reply:
Yes, I think it's critical from product perspective. (no crashes though)
This a new feature and without this change the UX will be bad in FF and Safari.

Reply:
I'll try to create a CL next week if nothing urgent happens.

Reply:
Let's set Major in https://youtrack.jetbrains.com/issue/CMP-7544